lib/exabgp/protocol/__init__.py

```diff
@@ -26,6 +26,11 @@ class Protocol (Enum):
 	PIM   = 0x67
 	SCTP  = 0x84
 
+	def __init__(self, value):
```

Comment:
The constructor should not raise: 
 - if the data comes from the wire, the size will be acceptable and we use it (it could unknown)
 - if the data comes from the configuration file, the value should be sanitised at input not in the class